components/components/app.js

```diff
@@ -0,0 +1,28 @@
+class App {
+  handleGetGradesError(error) {
+    console.log(error)
+  }
+  handleGetGradesSuccess(grades) {
+    this.gradeTable.updateGrades(grades) //pass its grades parameter to the updateGrades() method of the gradeTable property of the this object (instead of logging grades to the console).
+  }
+  constructor(gradeTable) {
```

Comment:
It doesn't make any difference to the computer, but when writing a class, you should make the `constructor` your first listed method if you have one, so that any fellow developers will know if that class uses a `constructor` method or not 👍